src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -473,30 +454,9 @@ private void init(long recordOffset) {
      * Initialize the client info (clientSequencer and offsetPersistedInSnowflake) by calling
      * ingestion service API configureClient and getClientStatus
      */
-    private void initClientInfoForExactlyOnceDelivery() {
+    private void initClientInfo() {
       ConfigureClientResponse configureClientResponse = ingestionService.configureClient();
       this.clientSequencer.set(configureClientResponse.getClientSequencer());
```

Comment:
we will not need this anymore. We will solely rely on Snowpipe's basic API which doesnt pass in client info.